setup.py

```diff
@@ -26,7 +26,7 @@
     # We use the ModelFormSetView from django-extra-views for the basket
```

Comment:
I think we should update this comment to reflect the new reason it is pinned.

Reply:
Yep, I've updated that comment.